### PR TITLE
fix predict result inserting

### PR DESCRIPTION
**Pull request by typhoonzero**
Refine inserting predict result to databases, so that for distributed database like hive, the input features data is the real source for the predict result, not from another "SELECT" statement.

reference: https://guillaumegenthial.github.io/serving-tensorflow-estimator.html

below todos should be finished before reviewing:

1. make sure calling `.predict` do not load weights every time
2. update code for keras too

### CODE REVIEW

**Comment by typhoonzero** [comment]
Wired that test case `TestEnd2EndMySQL/CaseTrainCustomModel` runs at local but will fail on CI.

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
 # Miniconda - Python 3.6, 64-bit, x86, latest
 ARG CONDA_ADD_PACKAGES=""
 ARG PIP_ADD_PACKAGES=""
-ARG TENSORFLOW_VERSION="2.0.0-b1"
+ARG TENSORFLOW_VERSION="2.0.0a0"
```
Comment (typhoonzero) [inline Dockerfile:13]: Will find out why keras model predict will hang at the end using `tensorflow==2.0.0b1`

File: sql/codegen.go [hunk 2]
```diff
@@ -410,62 +405,120 @@ def eval_input_fn(batch_size):
...
-            pred_rows.clear()
-    if len(pred_rows) > 0:
-        insert_cursor = insert_values(driver, write_conn, table_name, column_names, pred_rows)
+            next(self.predictions)
+        except Exception as e:
+            print("Exception in fast_predict. This is probably OK: %s" % e)
+
+column_names = feature_column_names[:]
+column_names.append("{{.Y.FeatureName}}")
+pred_gen = gen = db_generator(driver, conn, """{{.StandardSelect}}""", feature_column_names, "{{.Y.FeatureName}}", feature_metas)()
```
Comment (tonyyang-svail) [inline sql/codegen.go]: typo `pred_gen = gen = ...`?

**Review by tonyyang-svail — DISMISSED** [review]
LGTM. Maybe refactor the `While True` loop and `FastPredict` into `.py` file in the next PR.

File: Dockerfile (hunk 1 above)
Comment (tonyyang-svail) [reply Dockerfile:13]: For the record, maybe file an issue.

Comment (typhoonzero) [reply Dockerfile:13]: Added in https://github.com/sql-machine-learning/sqlflow/issues/573

File: sql/codegen.go [hunk 3]
```diff
@@ -415,60 +410,118 @@ def eval_input_fn(batch_size):
     dataset = dataset.map(ds_mapper).batch(batch_size)
     return dataset
 
-
-{{if .IsKerasModel}}
-pred_dataset = eval_input_fn(BATCHSIZE)
+pred_dataset = eval_input_fn(1)
```
Comment (Yancey0623) [inline sql/codegen.go:413]: comment here why hard code batchsize=1?

**Review by tonyyang-svail — APPROVED** [review]

File: sql/codegen.go (hunk 3 above)
Comment (typhoonzero) [reply sql/codegen.go:413]: Will add in next PR

**Pull request merged**